src/components/Callout/CalloutContent.tsx

```diff
@@ -52,9 +53,16 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
       slideDirectionalClassName: null,
       calloutElementRect: null
     };
+
+    if (typeof props.targetElement === 'string') {
+      this._targetElement = document.getElementById(props.targetElement);
```

Comment:
Hmmm... this is a bad idea on multiple levels.

1. You're caching the result in c'tor. If the targetElement value ever changes it never recomputes.
2. You're referring to "document" in ctor, which is violating the SSR requirement of not assuming you have a browser. I'm surprised this didn't break the SSR test...
3. You're using document... I recall there was a helper for getting document to support multiple document scenarios.